core/crypto/ecdsa/ecdsa_provider_impl.cpp

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "crypto/ecdsa/ecdsa_provider_impl.hpp"
+
+OUTCOME_CPP_DEFINE_CATEGORY(kagome::crypto, EcdsaProviderImpl::Error, e) {
+  using E = kagome::crypto::EcdsaProviderImpl::Error;
+  switch (e) {
+    case E::VERIFICATION_FAILED:
+      return "Internal error during ecdsa signature verification";
```

Comment:
VERIFICATION_FAILED kinda sounds like it is failed because the signature is wrong, I believe it'd be better to mention the internal error thing in the constant name.

Reply:
Also, where is it used?